packages/tailwind-preset/src/tailwind.preset.ts

```diff
@@ -285,10 +267,88 @@ const mantlePreset = {
 				unchecked: 'checked="false"',
 			},
 			backgroundColor: {
-				body: "hsl(var(--body) / <alpha-value>)",
-				background: "hsl(var(--background) / <alpha-value>)",
-				popover: "hsl(var(--popover) / <alpha-value>)",
-				card: "hsl(var(--card) / <alpha-value>)",
+				base: "hsl(var(--bg-base))",
```

Comment:
Pretty sure each of these needs an ` / <alpha-value.` in order for transparency to work, right?

Reply:
if we indeed want that control with these. I was wondering if we'd want these locked down. Not sure.

Reply:
You've got `bg-base/80` on the sheet backdrop and it seems to be working without these alpha value keys. Interesting!

Reply:
I think we should include the ` / <alpha-value>`

Reply:
I'm not certain about including `/ <alpha-value>` for these. Do we not want these values locked?

Reply:
> You've got `bg-base/80` on the sheet backdrop and it seems to be working without these alpha value keys. Interesting!

Oh strange. I've actually added a `bg-overlay` specifically for that now. I also opted for it to be darker in light mode as I think they do a better job drawing focus to the dialog.